intelmq/bots/BOTS

```diff
@@ -1005,6 +1006,9 @@
         "PostgreSQL": {
             "description": "PostgreSQL is the bot responsible to send events to a PostgreSQL Database. When activating autocommit, transactions are not used: http://initd.org/psycopg/docs/connection.html#connection.autocommit",
             "module": "intelmq.bots.outputs.postgresql.output",
+            "help": {
```

Comment:
I'm not sure if we really want to have the help for parameters in that file. (#757 proposes some major steps in how parameters are handled including types and docs/help.)

Reply:
Yeah, sorry, that's definitely not part of this PR, I've just wondered how to implement help description to the Manager and been designing some ways... Then I discovered #757 and other and saw that nothing is decided yet and gave up :D